frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -682,6 +695,11 @@ function ReaderHighlight:onTogglePanelZoomSetting(arg, ges)
     self.panel_zoom_enabled = not self.panel_zoom_enabled
 end
 
+function ReaderHighlight:onToggleFallbackTextSelection(arg, ges)
+    if not self.document.info.has_pages then return end
+    self.fallback_text_selection = not self.fallback_text_selection
```

Comment:
Mentionned that too quickly previously :)
I think all your "fallback text selection" should be renamed "fallback to text selection": that means: the variable name, this function :) because I would read "fallback text selection" as "another kind of text selection".
And given that we're in ReaderHighlight where everything (but panel zoom) is about text selection :), I would even name all your "fallback text selection" to "panel zoom fallback to text selection" so there's zero risk of confusion.

```lua
function ReaderHighlight:onTogglePanelZoomFallbackToTextSelection(arg, ges)
    self.panel_zoom_fallback_to_text_selection= not self.panel_zoom_fallback_to_text_selection
```